avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -660,15 +671,27 @@ def updatePositionC(cfg, particles, dem, force, DT):
 
     TotkinEneNew = TotkinEneNew + 0.5 * m * uMag * uMag
     TotpotEneNew = TotpotEneNew + mNew * gravAcc * zNew
+    TotForceSPHNew = TotForceSPHNew + mNew * norm(forceSPHX[j], forceSPHY[j], forceSPHZ[j])
```

Comment:
should we also put this in an if? this does not mean much for the real simulation

Reply:
done